### ISSUE TITLE
why concat the features and their inverse?

### ISSUE THREAD

**Issue by youth123** [tomiilab/DeepECA#1]
I am the new to bioinfos and confused with the feature operations. I don't understand the operations that concat the features and their inverse.
`#query	 query_=tf.to_float(tf.one_hot(tf.ones((n,n),dtype=tf.int32)*tf.cast(query,dtype=tf.int32),depth=20)) #n,20
	net=query_
	net=tf.concat([net,tf.transpose(query_,[1,0,2])],axis=2)
	#pssm
	pssm=tf.ones((n,n,21))*tf.expand_dims(fa,1) #(n,n,21)
	net=tf.concat([net,pssm],axis=2)
	net=tf.concat([net,tf.transpose(pssm,[1,0,2])],axis=2)
	#entropy
	entropy=tf.ones((n,n))*tf.reduce_sum(-fa*tf.log(fa),1) #n,n * n,21 -> n,n * n -> n,n
	net=tf.concat([net,tf.expand_dims(entropy,2)],axis=2)
	net=tf.concat([net,tf.expand_dims(tf.transpose(entropy),2)],axis=2)`
Please help me.
